# SPDX-License-Identifier: GPL-3.0-or-later
# Exact upstream install-only repair: 8a622b4b76d25d509ca50e8ca7fbf62d62753a4e.
# Archive29f and compiled registration/qmldir remain unchanged.
diff --git a/src/apps/settings/keyring/CMakeLists.txt b/src/apps/settings/keyring/CMakeLists.txt
index c9c8903bc..370035656 100644
--- a/src/apps/settings/keyring/CMakeLists.txt
+++ b/src/apps/settings/keyring/CMakeLists.txt
@@ -45,6 +45,6 @@ install(TARGETS qindaqt_settings_keyring_qml qindaqt_settings_keyring_qmlplugin
     LIBRARY DESTINATION "${QT6_INSTALL_QML}/QindaQt/SettingsApp/Keyring" COMPONENT SettingsAppearanceRuntime
     ARCHIVE DESTINATION "${QT6_INSTALL_QML}/QindaQt/SettingsApp/Keyring" COMPONENT SettingsAppearanceRuntime)
 install(FILES "${QINDAQT_KEYRING_SETTINGS_QML_OUTPUT_DIRECTORY}/qmldir"
-    "${QINDAQT_KEYRING_SETTINGS_QML_OUTPUT_DIRECTORY}/qindaqt_settings_keyring.qmltypes"
+    "${QINDAQT_KEYRING_SETTINGS_QML_OUTPUT_DIRECTORY}/qindaqt_settings_keyring_qml.qmltypes"
     qml/KeyringPage.qml
     DESTINATION "${QT6_INSTALL_QML}/QindaQt/SettingsApp/Keyring" COMPONENT SettingsAppearanceRuntime)
